src/Tizen.NUI/src/public/BaseComponents/View.cs

```diff
@@ -814,6 +814,7 @@ public Vector4 FlexMargin
         /// </summary>
         /// <remarks>
         /// The property cascade chaining set is possible. For example, this (view.CellIndex.X = 0.1f;) is possible.
+        /// Also, this property is for `TableView` class. Please use the property for the child position of TableView.
```

Comment:
I am not sure if it is correct, but can we use as follows?

```
/// Also, this property is for <see cref="TableView"/> class. Please use the property for the child position of <see cref="TableView"/>.
```

Reply:
Sure, we can!
Thanks for your review :)